config/default/forms/app/pregnancy.xml

```diff
@@ -995,7 +995,7 @@
       <bind nodeset="/pregnancy/tt_received" type="string" calculate="../safe_pregnancy_practices/tetanus/tt_imm_received"/>
       <bind nodeset="/pregnancy/t_pregnancy_follow_up_date" type="string" calculate="../anc_visits_hf/anc_visits_hf_next/anc_next_visit_date/appointment_date"/>
       <bind nodeset="/pregnancy/t_pregnancy_follow_up" type="string" calculate="../anc_visits_hf/anc_visits_hf_next/anc_next_visit_date/appointment_date_known"/>
-      <bind nodeset="/pregnancy/t_danger_signs_referral_follow_up_date" type="string" calculate="date-time(decimal-date-time(today()) + 3)"/>
+      <bind nodeset="/pregnancy/t_danger_signs_referral_follow_up_date" type="string" calculate="format-date-time(decimal-date-time(today()) + 3,&quot;%Y-%m-%d&quot;)"/>
```

Comment:
I am curious about what prompted this change?  What was wrong with relying on the default date serialization?  (Does not look like `t_danger_signs_referral_follow_up_date` is used anywhere else in the form). 

Also, minor, but I guess if we are not going to include any _time_ values, we can just use the `format-date` function instead of `format-date-time`....